guides/common/modules/proc_configuring-installation.adoc

```diff
@@ -51,6 +57,43 @@ endif::[]
 +
 The script displays its progress and writes logs to `{installer-log-file}`.
 
+ifdef::katello,satellite,orcharhino[]
+. Optional: If you want to use an external PostgreSQL database in your {Project} deployment, use the following command:
```

Comment:
Trying to avoid "following command":

```suggestion
. Optional: If you want to use an external PostgreSQL database in your {Project} deployment, configure your {ProjectServer}:
```

```suggestion
. Optional: Configure your {ProjectServer} to use an external PostgreSQL database:
```

I am unsure if my suggestions improve anything. Your call.